internal/serv/internal/auth/jwt.go

```diff
@@ -214,6 +219,8 @@ func firebaseKeyFunction(token *jwt.Token) (interface{}, error) {
 		firebasePublicKeys.Expiration = expiration
```

Comment:
Reading or writing to `firebasePublicKeys.Expiration` cannot be outside a lock. All fields inside a struct shared between go-routines need to be protected by a lock.

Reply:
Right. Should be fixed now.